README.md

```diff
@@ -11,8 +11,11 @@ A [hapi](https://github.com/hapijs/hapi) plugin integrating [Objection ORM](http
 Schwifty is used to define [Joi](https://github.com/hapijs/joi)-compatible models and knex connections for use with Objection ORM.  Those models then become available within your hapi server where it is most convenient.  It has been tailored to multi-plugin deployments, where each plugin may set clear boundaries in defining its own models, knex database connections, and migrations.  It's safe to register schwifty multiple times, wherever you'd like to use it, as it protects against model name collisions and other ambiguous configurations.
 
 ```js
-// First, ensure your project includes your
-// preferred versions of knex, objection, and sqlite3
+// First, ensure your project includes knex, objection, and sqlite3
+// Note that for knex and Objection, we assume using
+// knex >=0.8 and Objection >=1 <2 (see our peerDependencies in our package file)
+// Schwifty v4 introduced compatibility with Objection v.1
+// If you prefer / need to work with earlier versions of Objection, checkout Schwifty v3 instead!
 
 // To get started you might run,
 // npm install --save hapi@17 joi schwifty knex objection sqlite3
```

Comment:
We can get rid of `@17` now that it has been fully released.

Reply:
ahh gotcha. cool, thanks for catching that. i'll fix 'er up 🙏